### ISSUE TITLE
HTML Comments rendered with additional white space

### ISSUE THREAD

**Issue by gglnx** [mjmlio/mjml#2882]
**Describe the bug**
Currently, HTML comments get additional whitespace inside in the beginning and the end: `<!--#TEST-->` will be transformed to `<!-- #TEST -->`. CleverReach uses HTML comments as a programming language and doesn't support a space or any other whitespace character between the start of the comment and the command (`<!--#loop #-->` works, `<!-- #loop # -->` not).

**To Reproduce**
1. Create a file with this MJML code: `<mjml><!--#COMMAND#--></mjml>`
2. Render it to HTML and compare it

**Expected behavior**
Preserving of no or all whitespaces in HTML comments.

**MJML environment (please complete the following information):**
 - OS: MacOS
 - MJML Version: 4.15.3
 - MJML tool used: mjml-cli

**Comment by gglnx** [comment]
Yes, to temporarily fix it. But I would like to prefer it, that MJML doesn't modify the whitespace inside an HTML comment.

**Comment by iRyusa** [comment]
Might be fixable as the space might be introduced by this https://github.com/mjmlio/mjml/blob/master/packages/mjml-parser-xml/src/index.js#L315-L326

I'll check later
